From cf7d7ce38b8884ee4e2567ab047f90a96f2fb54b Mon Sep 17 00:00:00 2001
From: Divya Theja <divyathe@codeaurora.org>
Date: Mon, 11 Nov 2019 19:37:36 +0530
Subject: [PATCH] ptool.py: Generate zero files in output folder when
 specified.

The patch allows ptool to generate zerosector files in output directory
when such directory path is specified as argument to -t option.
---
 ptool.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ptool.py b/ptool.py
index 031ad74..66c95c5 100644
--- a/ptool.py
+++ b/ptool.py
@@ -362,7 +362,8 @@ def ShowBackupGPT(sector):
     print " "
 
 def CreateFileOfZeros(filename,num_sectors):
-
+    if OutputFolder:
+        filename = os.path.join(OutputFolder, filename)
     try:
         opfile = open(filename, "w+b")
     except Exception, x:
-- 
1.9.1

